### PR TITLE
feat: Define the spec Json-ld context URI

### LINKED ISSUE #17: Cleanup SSI namespaces
We need to adopt a namespace for custom attributes and cleanup use of the `ssi` prefix, preferably as `iatp` (Identity and Trust Protocol).

### PR DESCRIPTION
**Pull request by jimmarino**
Defines the spec Json-Ld context URI scheme used with versioning.

Linked Issue(s)
Closes #17

### CODE REVIEW

**Comment by jimmarino** [comment]
@borisrizov-zf

File: specifications/M1/credential.issuance.protocol.md [hunk 1]
```diff
@@ -173,14 +173,14 @@ The following is a non-normative example of a `credentials object`:
 {
   "format": "ldp_vc",
   "@context": {
-    "ssi": "https://w3id.org/edc/v0.8/",
+    "iatp": "https://w3id.org/tractusx-trust/v0.8",
     "odrl": "https://www.w3.org/ns/odrl.jsonld/",
```
Comment (wolf4ood) [inline specifications/M1/credential.issuance.protocol.md]:
```suggestion
    "odrl": "https://www.w3.org/ns/odrl/2/",
```

File: specifications/M1/identity.protocol.base.md [hunk 2]
```diff
@@ -109,3 +110,14 @@ scope of this specification.
 The Self-Issued ID Token request MAY contain the `bearer_access_scope` authorization request parameter which is set to a
 list of space-delimited scopes the response `VP Access Token` set in the `access_token` claim will be enabled for. If
 no `bearer_access_scope` parameter is present, the `access_token` claim MUST not be included.
+
+# 7. The Identity and Trust Protocol Context
+
+The `Json-ld context` URI for the all identity and trust specifications is:
+
+`https://w3id.org/tractusx-trust/v[version]`
```
Comment (wolf4ood) [inline specifications/M1/identity.protocol.base.md:118]: Does it make to use `iatp` also for the namespace and context urls?

Comment (jimmarino) [reply specifications/M1/identity.protocol.base.md:118]: That is optional for spec implementors. In the spec, yes we should use the same prefix, but this section is only about what implementations must do, not the spec iteself.

**Review by borisrizov-zf — APPROVED** [review]

**Review by wolf4ood — APPROVED** [review]

**Pull request merged**